Jenkinsfile

```diff
@@ -166,6 +166,16 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             }
         }
 
+        stage('Build Release Metadata') {
+            // Run the coreos-meta-translator against the most recent build,
+            // which will generate a release.json from the meta.json files
+            utils.shwrap("""
+            git clone https://github.com/coreos/fedora-coreos-releng-automation
```

Comment:
Minor: let's put this somewhere under e.g. `/var/tmp/` so as not to pollute the workdir? Doesn't matter now, though keeping a clear mental model of `$PWD = cosa workdir` is helpful.